sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/nimbus_recorded_targeting_context_v1/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+friendly_name: Nimbus Recorded Targeting Context (Fenix)
+description: |-
+  A query to obtain the previous day's recorded Nimbus targeting context values for each client.
+owners:
+- chumphreys@mozilla.com
+labels:
+  incremental: true
+scheduling:
+  dag_name: bqetl_default
+bigquery:
+  time_partitioning:
+    type: day
+    field: submission_date
+    require_partition_filter: true
+    expiration_days: null
```

Comment:
So I've confirmed that this setting will mean data does not expire, which I don't believe is intended or necessary here. I'll let you decide on a reasonable value.

Reply:
I'm going to set it to 400 so we'll have ~1yr of data